packages/x-charts/src/BarChart/BarLabel/BarLabel.tsx

```diff
@@ -28,7 +28,16 @@ export const BarLabelComponent = styled(animated.text, {
   },
 }));
 
-export type BarLabelProps = Omit<React.SVGProps<SVGTextElement>, 'ref' | 'id'> & BarLabelOwnerState;
+export type BarLabelProps = Omit<React.SVGProps<SVGTextElement>, 'ref' | 'id' | 'style'> &
+  BarLabelOwnerState & {
+    style: Omit<React.SVGProps<SVGTextElement>['style'], 'x' | 'y' | 'height' | 'width'> &
+      AnimatedProps<{
+        x: number;
+        y: number;
+        width: number;
+        height: number;
+      }>;
+  };
```

Comment:
I need to do this so that the types are correct, but then TypeScript fails because `useSlotProps` expects `style` to be `CSSProperties | undefined`. 

Should I move these properties from the `style` prop? IMO we should because they aren't spreadable to a normal HTML or SVG element. 

We're still in alpha, so we could still make a breaking change, couldn't we?

What do you think, @alexfauquette @JCQuintas?